src/component/BpmnVisualization.ts

```diff
@@ -45,6 +45,8 @@ export default class BpmnVisualization {
     try {
       const bpmnModel = newBpmnParser().parse(xml);
       defaultMxGraphRenderer(this.graph).render(bpmnModel, options?.fitType);
+      // eslint-disable-next-line no-console
+      console.log(this.graph);
```

Comment:
Is it normal to have a 'console.log' in the library ?

Reply:
no, I used it for test purpose, will remove it
thanks